src/proposal.mligo

```diff
@@ -337,6 +375,42 @@ let delete_proposal
     Set.remove (start_date, proposal_key) store.proposal_key_list_sort_by_date
   }
 
+let update_quorum(store, config : storage * config): storage =
```

Comment:
If picking the representation in the comment above here we should do only one thing, which is the first step from the comment above for `update_period_frozen_history` and that's it.